elastic/kibana
# [Response Ops][Alerting] Editing rule action does not auto-populate default summary message

Resolves https://github.com/elastic/kibana/issues/150715

## Summary

When editing an existing action on an existing rule, switching between the For each alert and Summary alert dropdown does switch the existing action message to the default message. 


### Checklist



### To verify

- create a rule with an action that uses `For each alert` notification frequency; use the default message
- edit your new rule and switch the action frequency to `Summary of alerts`; notice that the message does change to the default summary message
- create a rule with an action that uses `Summary of alerts` notification frequency; use the default message
- edit your new rule and switch the action frequency to `For each alert`; notice that the message does change to the default summary message
- repeat the steps above and but create the rule a with an action that does not use the default message
